### ISSUE TITLE
Expiacion WS also procing Tizona mana drain

### ISSUE THREAD

**Issue by thenoobers** [HomepointXI/Issues#599]
**Date & Time**: 

any

**Client Version (use `/ver` in game):**

30191204_1

**Character Name:**

noobers

**Discord Name (if different from in game):**

noobers

**Nation:**

bastok

**Job(level)/Sub Job(level):**

blu(99)rdm(49)

**NPC or Monster or item Name:**

any

**Zone name:** 

any

**Coordinates (use `!where`):**

any

**Item tool link (for items issues only):**

http://homepointxi.com/db/items/20652/tizona

**Multi-boxing? (multiple clients on same connection):**

yes or no

**Steps to reproduce:**

use alot of mana so you can tell, use Expiacion on literally any mob, the more dmg, the more mp.  i have gone from literally no mana to full when breaking 10kdmg.  Expiacion should not be procing the mana drain.

**Any additional information:**

**Comment by EpicTaru** [comment]
You are correct that it shouldn't be proccing on WS hits, it should only be proccing on normal hits, according to BG-wiki:

![image](https://user-images.githubusercontent.com/26195580/158029421-0598775a-841e-4da4-a89e-c31ec85c05e5.png)

Thank you for reporting the issue.  I can't tell you to not use Expiacion as it is a critical tool for usage of the Tizona weapon, but try not to overtly abuse this issue, please.
